HISTORY.rst

```diff
@@ -1,6 +1,45 @@
 =======
 History
 =======
+3.41.0 (2012-08-04)
+-------------------
+
+Changes
+
+* CCI has a new Metadata ETL task that allows adding fields to existing layouts. Sections can be selected using an index (#2766)
+
+* We added a task to enable an Einstein prediction (thanks, @erikperkins!) (#2778)
+
+* We added standard flows for releasing unlocked packages: release_unlocked_beta and release_unlocked_production (#2768)
+ 
+* We added documentation for using CumulusCI to build managed 2GP packages, 
+  unlocked packages, and extending NPSP and EDA with 2GP packages. (#2768)
+
+* Contributions to CCI now require verification by isort (#2770)
```

Comment:
I would leave this out of the release notes; it's not a customer-facing change.

Reply:
Unless you feel very strongly, I'm going to leave it (and clarify a bit). We get enough external contributors of custom tasks that I think it is of interest to people who would read these notes.

Reply:
@prescod I'm fine with including this sort of thing but I think we should put it in a separate section so that customers aren't scratching their heads trying to figure out if it's relevant to them.

Reply:
61061226398